cmd/ui/src/ducks/entityinfo/types.ts

```diff
@@ -94,6 +122,28 @@ export interface DomainInfo extends EntityInfo {
     containers: number;
 }
 
+// --- EnterpriseCA
+export interface EnterpriseCAInfo extends EntityInfo {
+    props: BasicInfo & {
+        basicconstraintpathlength: number;
+        caname: string;
+        casecuritycollected: boolean;
+        certchain: string[];
+        certname: string;
+        certthumbprint: string;
+        dnshostname: string;
+        enrollmentagentrestrictionscollected: boolean;
+        flags: 10;
```

Comment:
Is this supposed to be a single hard-coded value? Apologies in advance if it's business logic that I don't know yet! Im used to a literal type union (like  `10 | 20 | 30`), or just `number`.

Reply:
Good catch. I have updated this to `string` type to match the API's response for this entity.